torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -364,7 +364,7 @@ NvrtcFunction nvrtcCompile(
   const std::string compute = "--gpu-architecture=compute_" +
       std::to_string(major) + std::to_string(minor);
   std::vector<const char*> args = {
-      "--std=c++14", compute.c_str(), "-default-device"};
+      "--std=c++14", "--use_fast_math", compute.c_str(), "-default-device"};
```

Comment:
@jjsjann123 @kevinstephano any opinions on turning on fast math by default?

Reply:
No objection.

Just curious: "--use_fast_math" only changes flush denorm to zero. Is that a game changer in terms of perf?

Reply:
According to Deyu's Softmax experiments,

1. Baseline: ~200us
2. Outer Split - Coalesce Gmem: ~110us
3. Add fast-math compile flag: ~43us

Reply:
At least 10-15% improvement according to softmax_dropout benchmarks